2.1.c

```diff
@@ -0,0 +1,44 @@
+	
+#include <stdio.h>		
+
+//����������� ���������� ����� ��������� ����������(� ���������, ��� ��� n ~37 ��� ������� ��������), ����������� ����������, ������������� �������.
+
+
+int fibonacci(int n)
+{
+	if (n == 1)
+	{
+		return 1;
+	}
+	if (n == 0)
+	{
+		return 0;
+	}
+	else
+	{
+		return fibonacci(n - 1) + fibonacci(n - 2);
+	}
+
+}
+
+int antiFibonacci(int n)
+{
+	int a = 0, b = 1, c;

```

Comment:
```suggestion
	int a = 0, b = 1, c = 0;
```
Все переменные инициализируй при объявлении.